src/hooks/cgroups/pbs_cgroups.PY

```diff
@@ -63,14 +63,15 @@ This hook services the following events:
 # amount of resource requested will not be avaiable to the hook if they
 # are not present.
 
+# if we are not on a Linux system then the hook should always do nothing
+# and accept, and certainly not try Linux-only module imports
+import platform
+
```

Comment:
Why do we import this before pbs now?

Reply:
this was changed by my editor, which auto reorders imports... I don't think it will cause any problem in importing platform first instead pbs. but let me know if you think otherwise, I will change it.

Reply:
I just viewed it as an unnecessary change that didn't have anything to do with the renaming. I doubt it will cause a problem, but I was surprised to see it.